source/extensions/common/matcher/trie_matcher.cc

```diff
@@ -1,16 +1,24 @@
 #include "source/extensions/common/matcher/trie_matcher.h"
 
+#include "envoy/http/filter.h"
+#include "envoy/network/filter.h"
 #include "envoy/registry/registry.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace Common {
 namespace Matcher {
 
+class NetworkTrieMatcherFactory : public TrieMatcherFactoryBase<Network::MatchingData> {};
```

Comment:
Why should these class definitions be moved to `.cc`?

Reply:
For consistency. It doesn't seem to be useful to export these in a header.